AnnoDesigner/ViewModels/UpdateSettingsViewModel.cs

```diff
@@ -0,0 +1,309 @@
+                    IsPresetUpdateAvailable = true;
+                    FoundPresetRelease = foundRelease;
+                }
+            }
+        }
+
+        public ICommand CheckForUpdatesCommand { get; private set; }
+
+        private async void ExecuteCheckForUpdates(object param)
+        {
+            await CheckForUpdates(isAutomaticUpdateCheck: false);
+        }
+
+        public ICommand OpenReleasesCommand { get; private set; }
+
+        private void ExecuteOpenReleases(object param)
+        {
+            Process.Start("https://github.com/AnnoDesigner/anno-designer/releases");
+        }
+
+        public ICommand DownloadPresetsCommand { get; private set; }
+
+        private async void ExecuteDownloadPresets(object param)
+        {
+            if (FoundPresetRelease is null)
+            {
+                return;
+            }
+
+            BusyContent = Localization.Localization.Translations["UpdatePrefBusyDownloadPresets"];
+            IsBusy = true;
+
+            if (!_commons.CanWriteInFolder())
+            {
+                //already asked for admin rights?
+                if (Environment.GetCommandLineArgs().Any(x => x.Trim().Equals(Constants.Argument_Ask_For_Admin, StringComparison.OrdinalIgnoreCase)))
+                {
+                    MessageBox.Show($"You have no write access to the folder.{Environment.NewLine}The update can not be installed.",
+                        Localization.Localization.Translations["Error"],
+                        MessageBoxButton.OK,
+                        MessageBoxImage.Warning);
+
+                    IsBusy = false;
+                    return;
+                }
+
+                MessageBox.Show(Localization.Localization.Translations["UpdateRequiresAdminRightsMessage"],
+                    Localization.Localization.Translations["AdminRightsRequired"],
+                    MessageBoxButton.OK,
+                    MessageBoxImage.Information,
+                    MessageBoxResult.OK);
+
+                _appSettings.Save();
+                _commons.RestartApplication(true, Constants.Argument_Ask_For_Admin, App.ExecutablePath);
+            }
+
+            //Context is required here, do not use ConfigureAwait(false)
+            var newLocation = await _commons.UpdateHelper.DownloadReleaseAsync(FoundPresetRelease);
+            logger.Debug($"downloaded new preset ({FoundPresetRelease.Version}): {newLocation}");
+
+            IsBusy = false;
+
+            _appSettings.Save();
+            _commons.RestartApplication(false, null, App.ExecutablePath);
+
+            Environment.Exit(-1);
+        }
+
+        public bool AutomaticUpdateCheck
+        {
+            get { return _automaticUpdateCheck; }
+            set
+            {
+                if (UpdateProperty(ref _automaticUpdateCheck, value))
+                {
+                    _appSettings.Save();
+                }
+            }
+        }
+
+        public string VersionValue
```

Comment:
I wonder if its worth moving `VersionValue`, `FileVersionValue`, `PresetsVersionValue` to `Commons`? Then we wouldn't need to have this both here and in `MainViewModel`. If these were added to `Commons`, then we may not even need implement INotifyPropertyChanged on them.

Reply:
I think this is out of scope for this PR.
Maybe add as an issue?